### PR TITLE
Amended Vice-President per motion on 10/07/2016

### PR DESCRIPTION
**Pull request by ChadH1971**
Added text as voted upon by motion on 10/07/2016, made note of last date of amendment at the end of the document

### CODE REVIEW

**Comment by asterwolf** [comment]
I believe it looks good! I'll give @pathawks a chance to view all changes before merging through.

**Comment by ChadH1971** [comment]
Good idea.

File: pages/constitution.md [hunk 1]
```diff
@@ -194,5 +206,7 @@ Constitutions may also include a “Provision for By-Laws”; Article or a claus
 
 * The Web Page Submissions Committee shall be comprised of the officers who will select appropriate submissions for           inclusion  on the Student Organization Web Page. 
 * The officers shall be empowered to select a member to sit on the Web Page Submissions Committee. 
-* The time and place of regular meetings may be changed by vote of the membership. 
- 
+* The time and place of regular meetings may be changed by vote of the membership.
+
```
Comment (pathawks) [inline pages/constitution.md]: What changed on these two lines?

Comment (asterwolf) [inline pages/constitution.md]: I saw that as well. I just assumed it was just moved around; copied and pasted else where?

Comment (ChadH1971) [inline pages/constitution.md]: No change. When I added the final line, I didn't have a space between it and the line seperator so it was acting strange. It's the same line, but it was moved.

**Pull request merged**